test/common/export_results.c

```diff
@@ -0,0 +1,128 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2024 Nokia
+ */
+
+#include <odp_api.h>
+#include <odp/helper/odph_api.h>
+
+#include "export_results.h"
+
+#include <errno.h>
+#include <stdarg.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/types.h>
+#include <unistd.h>
+
+/** Maximum length of output path/name */
+#define MAX_FILENAME_LEN 192
+
+typedef struct {
+	test_common_options_t common_options;
+
+	char filename[MAX_FILENAME_LEN];
+
+	FILE *file;
+
+} test_export_gbl_t;
+
+static test_export_gbl_t gbl_data;
+
+int test_common_parse_options(int argc, char *argv[])
+{
+	char *env;
+	int i, j;
+
+	env = getenv("TEST_COMMON_EXPORT");
+	if (env) {
+		gbl_data.common_options.is_export = true;
+		odph_strcpy(gbl_data.filename, env, MAX_FILENAME_LEN);
+	}
+
+	/* Find and remove option */
+	for (i = 0; i < argc;) {
+		if (strcmp(argv[i], "--test-common-export") == 0) {
+			gbl_data.common_options.is_export = true;
+			if (i + 1 < argc && argv[i + 1][0] != '-') {
+				odph_strcpy(gbl_data.filename, argv[i + 1],
+					    MAX_FILENAME_LEN);
+				for (j = i; j < argc - 2; j++)
+					argv[j] = argv[j + 2];
+				argc -= 2;
+				continue;
+			} else {
+				for (j = i; j < argc - 1; j++)
+					argv[j] = argv[j + 1];
+				argc--;
+			}
+		}
+		i++;
+	}
```

Comment:
If `TEST_COMMON_EXPORT` is an empty string or `--test-common-export` doesn't have path argument the output file ends up being names `.csv`, which is likely not the intention.